test/cmd.js

```diff
@@ -335,35 +381,6 @@ describe('express(1)', function () {
       .expect(404, /<h1>Not Found<\/h1>/, done);
     });
   });
-
-  describe('--help', function () {
```

Comment:
And to be clear: I'm not exactly rejecting the other changes as a whole, but it does mean that in order to even accept the less tests, we have to finish discussion on the entire pull request, so all the time spent discussion the help stuff is all the time I have to continue to not accept the new less tests. Keeping them as separate tests allows for parallel discussions and the acceptance of the things separately. I hope that makes sense.